client/post-editor/editor-action-bar/style.scss

```diff
@@ -74,6 +74,10 @@
 	}
 }
 
+.is-nested-sidebar-focused .editor-action-bar {
```

Comment:
`is-modifier` classes should never be used without the component selector (example: `.button.is-primary {}`).